docs/requirements_for_rtd.txt

```diff
@@ -1,73 +1,73 @@
-Flask-Cache==0.13.1
-Flask-Cors==3.0.7
-Flask-HTTPAuth==3.2.4
-Flask-RESTful==0.3.7
-Flask-SQLAlchemy==2.3.2
-Flask==1.0.2
-Jinja2==2.10.1
-MarkupSafe==1.1.1
 PyCifRW==4.2.1; python_version < '3'
 PyCifRW==4.4; python_version >= '3'
-PyMySQL==0.9.3
-PyYAML==3.13
-Pygments==2.3.1
-SQLAlchemy-Utils==0.33.11
-SQLAlchemy==1.3.3
-Sphinx==1.8.4
 aiida-export-migration-tests==0.7.0
 aldjemy==0.9.1
-alembic==1.0.7
+alembic==1.2.1
 ase==3.17.0
 circus==0.15.0
 click-completion==0.5.1
 click-config-file==0.5.0
 click-spinner==0.1.8
 click==7.0
 codecov==2.0.15
-coverage==4.5.2
-django==1.11.20
-docutils==0.14
+coverage==4.5.4
+django==1.11.25
+docutils==0.15.2
 enum34==1.1.6; python_version<'3.5'
 ete3==3.1.1
-flask-marshmallow==0.9.0
+flask-cache==0.13.1
+flask-cors==3.0.8
+flask-httpauth==3.3.0
+flask-marshmallow==0.10.1
+flask-restful==0.3.7
+flask-sqlalchemy==2.4.1
+flask==1.1.1
```

Comment:
Shouldn't these explicitly be `Flask` and not `flask`?

Reply:
I decided for simplicity to convert everything to lowercase. I think pip is capital insensitive.

Reply:
> I think pip is capital insensitive.

Yeah, I couldn't find anything on PyPI that says it is - there was only something about case sensitivity for entry points in pip.